storage/migrations/01_oasis_3_consensus.up.sql

```diff
@@ -36,8 +36,8 @@ CREATE TABLE oasis_3.transactions
 (
   block BIGINT NOT NULL REFERENCES oasis_3.blocks(height) DEFERRABLE INITIALLY DEFERRED,
 
-  txn_hash   HEX64 NOT NULL,
-  txn_index  UINT31,
+  txn_hash   HEX64,
```

Comment:
We could keep this as `txn_hash HEX64 NOT NULL`?

Reply:
yes please